apps/pdl-forvalter/src/main/java/no/nav/pdl/forvalter/service/IdenttypeService.java

```diff
@@ -54,64 +56,18 @@ public class IdenttypeService implements Validation<IdentRequestDTO> {
     private final MapperFacade mapperFacade;
     private final PersonRepository personRepository;
 
-    private static Identtype getIdenttype(IdentRequestDTO request, String ident) {
-
-        if (nonNull(request.getIdenttype())) {
-            return request.getIdenttype();
-        }
-        return nonNull(ident) ? IdenttypeFraIdentUtility.getIdenttype(ident) : FNR;
-    }
-
-    private static Kjoenn getTilfeldigKjoenn() {
-
-        return secureRandom.nextBoolean() ? MANN : KVINNE;
-    }
-
-    private static Kjoenn getKjoenn(IdentRequestDTO request, String ident) {
-
-        if (nonNull(request.getKjoenn()) && request.getKjoenn() != UKJENT) {
-            return request.getKjoenn();
-        }
-        return nonNull(ident) ? KjoennFraIdentUtility.getKjoenn(ident) : getTilfeldigKjoenn();
-    }
-
-    private static LocalDateTime getFoedtFoer(IdentRequestDTO request, String ident) {
-
-        if (nonNull(request.getFoedtFoer())) {
-            return request.getFoedtFoer();
-        } else if (nonNull(request.getAlder())) {
-            return LocalDateTime.now().minusYears(request.getAlder()).minusMonths(3);
-        }
-        return (nonNull(ident)) ? DatoFraIdentUtility.getDato(ident).plusMonths(1).atStartOfDay() :
-                LocalDateTime.now().minusYears(18);
-    }
-
-    private static LocalDateTime getFoedtEtter(IdentRequestDTO request, String ident) {
-
-        if (nonNull(request.getFoedtEtter())) {
-            return request.getFoedtEtter();
-        } else if (nonNull(request.getAlder())) {
-            return LocalDateTime.now().minusYears(request.getAlder()).minusYears(1);
-        }
-        return (nonNull(ident)) ? DatoFraIdentUtility.getDato(ident).minusMonths(1).atStartOfDay() :
-                LocalDateTime.now().minusYears(67);
-    }
-
-    private static boolean isSyntetisk(IdentRequestDTO request, String ident) {
-        return nonNull(request.getSyntetisk()) ? request.getSyntetisk() :
-                SyntetiskFraIdentUtility.isSyntetisk(ident);
-    }
-
     public PersonDTO convert(PersonDTO person) {
 
         var nyPerson = person;
+        log.info("Gjeldende ident: {}", person);
         for (var type : person.getNyident()) {
 
             if (isTrue(type.getIsNew())) {
 
                 nyPerson = handle(type, nyPerson);
                 type.setKilde(isNotBlank(type.getKilde()) ? type.getKilde() : "Dolly");
                 type.setMaster(nonNull(type.getMaster()) ? type.getMaster() : DbVersjonDTO.Master.FREG);
+                type.setFoedtEtter(person.getFoedsel().getFirst().getFoedselsdato().plusDays(1));
```

Comment:
To mulige problemer:
1. Har du testet denne hvis nyIdent bestilles på opprinnelig bestilling, det går nok bra da denne klasse kalles etter FoedselsService, [https://github.com/navikt/testnorge/blob/75a58da7a6c670cbb620fc51c8c76b43962b6612/apps/pdl-forvalter/src/main/java/no/nav/pdl/forvalter/service/PersonArtifactService.java], så OK
2. Hvis det er en liste flere nye identer inn, det kan vel skje men kanskje ikke så ofte, da er ikke utfallet entydig.